### PR TITLE
[RNG] Returning tests

### PR DESCRIPTION
**Pull request by kcenia4010**
Previous tests are returned

### CODE REVIEW

File: test/rng_testsuite/statistics_tests/bernoulli_distribution_test.pass.cpp [hunk 1]
```diff
@@ -178,95 +192,189 @@ main()
...
+    err += tests_set_portion<sycl::vec<bool, 16>, sycl::vec<std::uint32_t, 2>>(queue, 140, 7);
+    err += tests_set_portion<sycl::vec<bool, 16>, sycl::vec<std::uint32_t, 1>>(queue, 140, 7);
+    err += tests_set_portion<sycl::vec<bool, 16>, sycl::vec<std::uint32_t, 16>>(queue, 160, 17);
+    err += tests_set_portion<sycl::vec<bool, 16>, sycl::vec<std::uint32_t, 8>>(queue, 160, 17);
+    err += tests_set_portion<sycl::vec<bool, 16>, sycl::vec<std::uint32_t, 4>>(queue, 160, 17);
+    err += tests_set_portion<sycl::vec<bool, 16>, sycl::vec<std::uint32_t, 3>>(queue, 160, 17);
+    err += tests_set_portion<sycl::vec<bool, 16>, sycl::vec<std::uint32_t, 2>>(queue, 160, 17);
+    err += tests_set_portion<sycl::vec<bool, 16>, sycl::vec<std::uint32_t, 1>>(queue, 160, 17);
 #endif // TEST_LONG_RUN
-    EXPECT_TRUE(!err, "Test FAILED");
```
Comment (aelizaro) [inline test/rng_testsuite/statistics_tests/bernoulli_distribution_test.pass.cpp:269]: why line 269 is deleted?

File: test/rng_testsuite/statistics_tests/exponential_distribution_test.pass.cpp [hunk 2]
```diff
@@ -282,80 +390,190 @@ main()
...
+    err += tests_set_portion<sycl::vec<double, 16>, sycl::vec<std::uint32_t, 2>>(queue, 140, 7);
+    err += tests_set_portion<sycl::vec<double, 16>, sycl::vec<std::uint32_t, 1>>(queue, 140, 7);
+    err += tests_set_portion<sycl::vec<double, 16>, sycl::vec<std::uint32_t, 16>>(queue, 160, 17);
+    err += tests_set_portion<sycl::vec<double, 16>, sycl::vec<std::uint32_t, 8>>(queue, 160, 17);
+    err += tests_set_portion<sycl::vec<double, 16>, sycl::vec<std::uint32_t, 4>>(queue, 160, 17);
+    err += tests_set_portion<sycl::vec<double, 16>, sycl::vec<std::uint32_t, 3>>(queue, 160, 17);
+    err += tests_set_portion<sycl::vec<double, 16>, sycl::vec<std::uint32_t, 2>>(queue, 160, 17);
+    err += tests_set_portion<sycl::vec<double, 16>, sycl::vec<std::uint32_t, 1>>(queue, 160, 17);
 #endif // TEST_LONG_RUN
-    EXPECT_TRUE(!err, "Test FAILED");
```
Comment (aelizaro) [inline test/rng_testsuite/statistics_tests/exponential_distribution_test.pass.cpp:358]: same question here, line 358 is deleted, we don't check err here

File: test/rng_testsuite/statistics_tests/bernoulli_distribution_test.pass.cpp (hunk 1 above)
Comment (kcenia4010) [reply test/rng_testsuite/statistics_tests/bernoulli_distribution_test.pass.cpp:269]: done

File: test/rng_testsuite/statistics_tests/exponential_distribution_test.pass.cpp (hunk 2 above)
Comment (kcenia4010) [reply test/rng_testsuite/statistics_tests/exponential_distribution_test.pass.cpp:358]: done

**Pull request merged**